azure-pipelines.yml

```diff
@@ -29,27 +33,34 @@ variables:
   buildConfiguration: Release
   dotNetCoreRuntimeIdentifier: win-x64 
   dotNetCoreVersion: '3.1'
-  dotNetCorePublishArtifactPath: 'DotNetCorePublish'
-  dotNetCorePublishArtifactName: 'DotNetCoreExecutables'
 
   # Sample 1: Windows Voice Assistant Client, C# WPF (\clients\csharp-wpf)
   sample1Name: VoiceAssistantClient
   sample1Folder: 'clients\csharp-wpf'
   sample1Solution: '$(sample1Folder)\$(sample1Name).sln'
   sample1PublishedArtifactPath: '$(sample1Folder)\$(sample1Name)\bin\$(buildPlatformNative)\$(buildConfiguration)'
-  sample1PublishedArtifactName: 'WindowsVoiceAssistantClientBuild'
+  sample1PublishedArtifactName: 'WindowsVoiceAssistantClient-$(Build.BuildNumber)'
 
   # Sample 2: Voice Assistant Test, .NET core (\clients\csharp-dotnet-core\voice-assistant-test)
   sample2Name: VoiceAssistantTest
   sample2Folder: 'clients\csharp-dotnet-core\voice-assistant-test\tool'
   sample2Solution: '$(sample2Folder)\$(sample2Name).sln'
-  
+  sample2PublishedArtifactPath: 'DotNetCorePublish'
```

Comment:
Should this be a path similar to sample1, 3, and 4?

Reply:
@splumber Thanks, as you know this is a temporary folder that is created to hold the result of the .NET Core publish command. I made another commit to change the folder name to something unique to this sample.